redirects-transition.conf

```diff
@@ -1,5 +1,9 @@
 #Redirect rules for specific pages
 
+rewrite ^/pdf/CoverLetter_20130725.pdf https://www.fec.gov/resources/cms-content/documents/letter_to_Committee_on_House_Administration_July_25_2013.pdf redirect;
+rewrite ^/pdf/Additional_Enforcement_Materials.pdf https://www.fec.gov/resources/cms-content/documents/additional_enforcement_materials.pdf redirect;
+rewrite ^/pdf/1997_Enforcement_Manual.pdf https://www.fec.gov/resources/cms-content/documents/1997_enforcement_manual.pdf redirect;
+rewrite ^/pages/brochures/spec_notice_brochure.pdf https://www.fec.gov/help-candidates-and-committees/advertising-and-disclaimers redirect;
```

Comment:
Please add a `/` at the end of the redirect request. It should look like this:

`rewrite ^/pages/brochures/spec_notice_brochure.pdf https://www.fec.gov/help-candidates-and-committees/advertising-and-disclaimers/ redirect;`

Reason for this is because the website does another redirect if yo don't have the slash at the end of webpage URLs, so this helps to prevent that extra hop that's unneeded. Please do this for line 45 changes as well.